ee_plugin/ee_plugin.py

```diff
@@ -112,36 +112,54 @@ def initGui(self):
             parent=self.iface.mainWindow(),
             triggered=self._run_cmd_set_cloud_project,
         )
+        add_fc_button = QtWidgets.QAction(
```

Comment:
Nice! I'm wondering if when we should implement the sub-menus to avoid having a long list of functions at the top level. We can punt beyond this PR.

Reply:
> I'm wondering if when we should implement the sub-menus to ...

Not sure what you're saying here.

Reply:
Basically, have an "EE Tools" or something like at the top level and list the specific forms available under this sub-menu. In a similar way to how other menus work in QGIS: 
<img width="470" alt="image" src="https://github.com/user-attachments/assets/c95191d6-278f-4ebb-a45e-b7234f0b2f68" />

Reply:
<img width="759" alt="image" src="https://github.com/user-attachments/assets/99e05452-60bb-4c31-9ea1-fe335a9e267e" />
How does this look to you?